rln/src/iden3calc.rs

```diff
@@ -0,0 +1,73 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
```

Comment:
What the reason oh having this one here? Why not fix names regarding linter?

Reply:
Ok, I will remove this, since I do not know the motivation behind this.